### PR TITLE
fix(tutorials): wordpress & lemp stack MTA-1593

### PR DESCRIPTION
**Pull request by RoRoJ**
- Update/validation of
     - tutorials/installation-lemp-ubuntu-bionic
     - tutorials/wordpress-lemp-stack (also for Bionic)

- Creation of new versions for Focal Fossa:
    - tutorials/installation-lemp-ubuntu-focal
     - tutorials/wordpress-lemp-stack-focal

### CODE REVIEW

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  title: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic
-  description: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic (18.04)
+  title: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver
+  description: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver (18.04)
 content:
-  h1: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic
-  paragraph: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic (18.04)
-tags: account LEMP-Stack Ubuntu-Bionic 
+  h1: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver
+  paragraph: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on UbuntuBionic Beaver (18.04)
```
Comment (bene2k1) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:7]:
```suggestion
  paragraph: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver (18.04)
```

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx [hunk 2]
```diff
@@ -124,38 +127,45 @@ The [LEMP-Stack](https://lemp.io/) references a group of software that can be us
...
-                    fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
-            }
-
-            location ~ /\.ht {
-                    deny all;
-            }
-    }
+          listen 80;
+          root /var/www/html;
+          index index.php index.html index.htm index.nginx-debian.html;
```
Comment (bene2k1) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:148]:
```suggestion
          index index.php index.html index.htm;
```
I don't think you need the last file, it refers to the Debian specific index file only

File: tutorials/installation-lemp-ubuntu-focal/index.mdx [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  cd /etc/nginx/sites-available/
+  ```
+
+2. Create a new server block configuration file and name it after the domain name that points to your Instance. Here we call it `example.com` and put the following content into the file:
+
+  ```
+    server {
+          listen 80;
+          root /var/www/html;
+          index index.php index.html index.htm index.nginx-debian.html;
```
Comment (bene2k1) [inline tutorials/installation-lemp-ubuntu-focal/index.mdx:148]:
```suggestion
          index index.php index.html index.htm;
```
same as above

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 4]
```diff
@@ -74,11 +75,11 @@ You will now need to configure Nginx to deliver traffic to the future WordPress
             root /var/www/wordpress;
             index index.php index.html index.htm;
 
-            server_name blog.scw-site.ml;
+            server_name blog.example.com;
 
-            error_page 404 /404.html;
+            #error_page 404 /404.html;
```
Comment (bene2k1) [inline tutorials/wordpress-lemp-stack/index.mdx:80]:
```suggestion
```
Remove this line as it is commented-out and no required

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 5]
```diff
@@ -74,11 +75,11 @@ You will now need to configure Nginx to deliver traffic to the future WordPress
...
 
-            error_page 500 502 503 504 /50x.html;
+            #error_page 500 502 503 504 /50x.html;
```
Comment (bene2k1) [inline tutorials/wordpress-lemp-stack/index.mdx:82]:
```suggestion
```
Remove this line as it is commented-out and not required

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 6]
```diff
@@ -90,7 +91,8 @@ You will now need to configure Nginx to deliver traffic to the future WordPress
             location ~ \.php$ {
                     try_files $uri =404;
                     fastcgi_split_path_info ^(.+\.php)(/.+)$;
-                    fastcgi_pass unix:/var/run/php5-fpm.sock;
+                    #fastcgi_pass unix:/var/run/php7.2-fpm.sock;
```
Comment (bene2k1) [inline tutorials/wordpress-lemp-stack/index.mdx:94]:
```suggestion
```
Remove this line as it is commented-out and not required

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx [hunk 7]
```diff
@@ -111,7 +114,7 @@ The [LEMP-Stack](https://lemp.io/) references a group of software that can be us
   mysql_secure_installation
   ```
 
-2. You will be asked to enter the current root password for the MariaDB server. As it not yet set, press `Enter`.
+2. Enter the current root password for the MariaDB server. when prompted As it not yet set, press `Enter`.
```
Comment (ofranc) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:117]:
```suggestion
2. Enter the current root password for the MariaDB server. When prompted, as it is not yet set, press `Enter`.
```

File: tutorials/installation-lemp-ubuntu-focal/index.mdx [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+
+## Configuring MySQL / MariaDB
+
+1. Launch the configuration assistant for the database server:
+
+  ```bash
+  mysql_secure_installation
+  ```
+
+2. Enter the current root password for the MariaDB server. when prompted As it not yet set, press `Enter`.
```
Comment (ofranc) [inline tutorials/installation-lemp-ubuntu-focal/index.mdx:117]:
```suggestion
2. Enter the current root password for the MariaDB server. When prompted, as it is not yet set, press `Enter`.
```

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 9]
```diff
@@ -29,43 +29,44 @@ categories:
...
 - You have [LEMP installed](/tutorials/installation-lemp-ubuntu-bionic/)
+- You have a FQDN (Fully Qualified Domain Name) pointing to your Instance’s IP address
 - You have [secured your LEMP stack with TLS/SSL](/tutorials/installation-lemp-ubuntu-bionic/#configuring-ssl-with-let's-encrypt)
 
 </Message>
 
 ## Installing the stack
 
-As a tutorial on LEMP is already available, you can follow the procedure at [Installing LEMP on Ubuntu Bionic.](/tutorials/installation-lemp-ubuntu-bionic/)
+As a tutorial on LEMP is already available, you can follow the procedure at [Installing LEMP on Ubuntu Bioic Beaver.](/tutorials/installation-lemp-ubuntu/)
```
Comment (ofranc) [inline tutorials/wordpress-lemp-stack/index.mdx:55]:
```suggestion
As a tutorial on LEMP is already available, you can follow the procedure at [Installing LEMP on Ubuntu Bionic Beaver.](/tutorials/installation-lemp-ubuntu/)
```

**Review by ofranc — APPROVED** [review]

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx [hunk 10]
```diff
@@ -124,38 +127,45 @@ The [LEMP-Stack](https://lemp.io/) references a group of software that can be us
...
+          }
+
+          location ~ /\.ht {
+                  deny all;
+          }
+  }
   ```
 
-  Overview:
+  Remember to replace `example.com` and any other information as necessary as per the overview below:
```
Comment (ldecarvalho-doc) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:166]:
```suggestion
  Remember to replace `example.com` and any other information necessary as per the overview below:
```

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx [hunk 11]
```diff
@@ -198,7 +210,7 @@ Nginx stores the configuration of virtual hosts in server blocks. All available
   ?>
   ```
 
-3. Save the file and point your web browser to `http://YOUR_DOMAIN_NAME.TLD/phpinfo.php`. When you see an output like the following, PHP is setup correctly:
+3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, PHP is setup correctly:
```
Comment (ldecarvalho-doc) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:213]:
```suggestion
3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, it means PHP is set up correctly:
```

File: tutorials/installation-lemp-ubuntu-focal/index.mdx [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+  Set root password? [Y/n]
+  ```
+
+4. Enter the new password and press `Enter`, then repeat this step.
+
+5. Keep the settings proposed by default when prompted by further questions, by pressing `Enter` each time.
+
+## Configuring a Nginx Server-Block
+
+Nginx stores the configuration of virtual hosts in server blocks. All available server blocks are located in the directory `/etc/nginx/sites-available/`.
```
Comment (ldecarvalho-doc) [inline tutorials/installation-lemp-ubuntu-focal/index.mdx:134]:
```suggestion
Nginx stores the configuration of virtual hosts in server blocks. All available server blocks are located in the `/etc/nginx/sites-available/` directory.
```

File: tutorials/installation-lemp-ubuntu-focal/index.mdx [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+
+2. Put the following content in it:
+
+  ```php
+  <?php
+  phpinfo();
+  ?>
+  ```
+
+3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, PHP is setup correctly:
```
Comment (ldecarvalho-doc) [inline tutorials/installation-lemp-ubuntu-focal/index.mdx:213]:
```suggestion
3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, it means PHP is set up correctly:
```

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 14]
```diff
@@ -29,43 +29,44 @@ categories:
...
+WordPress is a popular, free open-source blogging tool and content management system (CMS) based on PHP and MySQL. WordPress has seen incredible adoption rates, and is a great choice for getting a website up and running quickly. After setup, almost all the administration can be done through the web frontend.
 
-In this tutorial, we will learn how to install WordPress on a freshly created Ubuntu Bionic instance with LEMP (Linux + Nginx - pronounced "engine x" + MySQL + PHP). Compared to Apache, Nginx is a HTTP server that uses much less resources and delivers pages a lot of faster, especially static files.
+In this tutorial, we will learn how to install WordPress on a freshly created Ubuntu Bionic Beaver instance with LEMP (Linux + Nginx - pronounced "engine x" + MySQL + PHP). Nginx is a HTTP server that, compared to Apache, uses less resources and delivers pages much faster, especially static files.
```
Comment (ldecarvalho-doc) [inline tutorials/wordpress-lemp-stack/index.mdx:40]:
```suggestion
In this tutorial, we will learn how to install WordPress on a freshly created Ubuntu Bionic Beaver instance with LEMP (Linux + Nginx - pronounced "engine x" + MySQL + PHP). Nginx is an HTTP server that, compared to Apache, uses less resources and delivers pages much faster, especially static files.
```

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 15]
```diff
@@ -158,17 +157,19 @@ You will now need to configure Nginx to deliver traffic to the future WordPress
...
   nano /etc/nginx/nginx.conf
   ```
 
-7. Edit the following:
+7. Edit the following to optimize:
 
     <Message type="tip">
 
-    Ensure the number of worker processes equals to the number of cores in your instance
+    Ensure the number of worker processes equals to the number of cores in your Instance
```
Comment (ldecarvalho-doc) [inline tutorials/wordpress-lemp-stack/index.mdx:172]:
```suggestion
    Ensure the number of worker processes is equal to the number of cores in your Instance.
```

File: tutorials/wordpress-lemp-stack/index.mdx [hunk 16]
```diff
@@ -343,20 +367,47 @@ In your web browser, navigate to your server's domain name or public IP address:
 
 1. Select your language.
 
-2. Enter the information in relevance with your account.
+  You are asked to enter your database information. Before doing so, you need to go back to the terminal to complete the next steps.
```
Comment (ldecarvalho-doc) [inline tutorials/wordpress-lemp-stack/index.mdx:370]:
```suggestion
  You are asked to enter your database information. Before doing so, go back to the terminal to complete the next steps.
```

File: tutorials/installation-lemp-ubuntu-bionic/index.mdx (hunk 7 above)
Comment (RoRoJ) [inline tutorials/installation-lemp-ubuntu-bionic/index.mdx:117]:
```suggestion
2. Press "Enter" when prompted for the current root password for the MariaDB server, as the password is not yet set.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**